src/main/webapp/populateData.js

```diff
@@ -104,19 +116,29 @@ function populateTasks() {
         return response.json();
       })
       .then((tasksResponse) => {
+        if (options.length === 0) {
```

Comment:
Why not update the set of task lists every time?

Reply:
There's a double call of `handleAuthenticationState` so this is just a workaround - updating the set of task lists every time would mean there's two copies of the same task lists. Here's the [discussion](https://github.com/googleinterns/step7-2020/pull/114#discussion_r460953024) for more context.